### PR TITLE
Merge berkeley develop 28 08 2023

### PR DESCRIPTION
**Pull request by georgeee**
Merge `berkeley` to `develop`.

### CODE REVIEW

**Review by Trivo25 — APPROVED** [review]

**Review by ghost-not-in-the-shell — APPROVED** [review]

**Review by QuiteStochastic — APPROVED** [review]

**Review by bkase — APPROVED** [review]
Codeowners 👍

File: buildkite/src/Jobs/Lint/Merge.dhall [hunk 1]
```diff
@@ -88,6 +88,50 @@ Pipeline.build
...
+        Command.Config::{
+          commands = [ Cmd.run "scripts/merged-to-proof-systems.sh develop"]
+          , label = "[proof-systems] Check merges cleanly into proof-systems develop branch"
+          , key = "merged-to-proof-systems-develop"
+          , soft_fail = Some (B/SoftFail.Boolean True)
+          , target = Size.Small
+          , docker = Some Docker::{
+              image = (../../Constants/ContainerImages.dhall).toolchainBase
+            }
+        },
```
Comment (mitschabaude) [inline buildkite/src/Jobs/Lint/Merge.dhall:134]: these changes are actually duplicated code due to manual rebasing, you have to delete them to get CI to work

**Comment by georgeee** [comment]
_@Trivo25 to merge the latest `berkeley` to this PR, as there are some conflicts_

File: buildkite/src/Jobs/Lint/Merge.dhall (hunk 1 above)
Comment (mitschabaude) [reply buildkite/src/Jobs/Lint/Merge.dhall]: lgtm!

**Comment by georgeee** [comment]
Merge completed in another PR

**Pull request closed without merging**